17/windows/windowsservercore-ltsc2019/Dockerfile

```diff
@@ -24,18 +24,8 @@
 ARG version=3107.v665000b_51092-5
 FROM jenkins/agent:${version}-jdk17-windowsservercore-ltsc2019
 
+ARG version=3107.v665000b_51092-5
```

Comment:
```suggestion
# Use the instruction in the current scope as per https://docs.docker.com/engine/reference/builder/#scope (requires duplication of the "before-FROM" ARG)
ARG version=3107.v665000b_51092-5
```

WDYT @timja ^^ ?

Reply:
Could we just drop the arg? Seems to be only used in a version label that should be set on the parent?

Reply:
Yes i agree, but that would require a full change on all images.

I don’t mind removing it from this PR either as it’s out of scope on both cases

Reply:
I’ll open a PR with the arg + label removal on all images